modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -4136,6 +4139,23 @@ private CompletableFuture<?> processVacuumTxStateReplicaRequest(VacuumTxStateRep
         return raftClient.run(cmd);
     }
 
+    private CompletableFuture<?> processMinimumActiveTxTimeReplicaRequest(UpdateMinimumActiveTxBeginTimeReplicaRequest request) {
+        Command cmd = PARTITION_REPLICATION_MESSAGES_FACTORY.updateMinimumActiveTxBeginTimeCommand()
+                .timestamp(request.timestamp())
+                .build();
+
+        CompletableFuture<Object> resultFuture = new CompletableFuture<>();
+
+        // The timestamp must increase monotonically, otherwise it will have to be
```

Comment:
In that case you will need to extend SafeTimePropagatingCommand instead of simple WriteCommand and specify safe time of course.